src/index.html

```diff
@@ -9,10 +9,54 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/main.css"
```

Comment:
Ensure that the path to the CSS file (`/src/styles/main.css`) is correct and that the file exists. This link is crucial for applying styles to the calendar.